lib/rvg/rvg.rb

```diff
@@ -60,10 +60,10 @@ class RVG
     # else, combine it with the background_fill_opacity.
     def bgfill
       if @background_fill.nil?
-        color = Magick::Pixel.new(0, 0, 0, Magick::TransparentOpacity)
+        color = Magick::Pixel.new(0, 0, 0, Magick::QuantumRange)
```

Comment:
Probably better to use `OpaqueAlpha` here instead of `QuantumRange`.

Reply:
Thanks. fixed with https://github.com/rmagick/rmagick/pull/756/commits/9de14c0715be434062755e24174c2a9bc43ab2ff

Reply:
And sorry for the unfriendly block of the PR. I really wish there was a some kind of please change this instead of you must change this.

Reply:
No worries. Thank you for your reviewing 😆